programming/Encoders.md

```diff
@@ -0,0 +1,53 @@
+# Encoders
+Ever been on a website written in another language and had to utilize google translate to gain some 
+semblance of recognition as to what it was you were looking at?
+Encoders work in a similar fashion to Google Translate, of course instead of translating languages Encoders translate
```

Comment:
Please remove references to this kind of encoders. We **only** want to include references to the rotation sensor definition of encoder.